src/medium-zoom.js

```diff
@@ -13,6 +15,23 @@ const isListOrCollection = selector =>
 
 const isNode = selector => selector && selector.nodeType === 1
 
+/**
```

Comment:
Can we not include the polyfill in the source code? Perhaps document in the README to include this polyfill before including `medium-zoom` for IE support. What do you think?

Reply:
Yep. Agreed on not including the polyfill as-is. Libs shouldn't mess with globals (that's my bad).

If it were my choice, I'd switch from a CustomEvent polyfill (which modifies the `window` object) to a [ponyyfill](https://github.com/sindresorhus/ponyfill). The rational for this is so that libraries that depend on medium-zoom aren't *forced* to polyfill to provide support for older browsers. If medium-zoom requires ponyfills to function, then so do all libraries that depend on it (unless they use a forked version of medium-zoom and do the ponyfilling themselves but... that kinda stinks). In this case, we can switch the current CurrentEvent polyfill to a ~6-line ponyfill very easily and prevent devs from having to deal the issue which seems like the right thing to do.

For `template` support, I did a quick scan of the code and it looks like we could get this working with minimal changes. I haven't spend enough time looking into this to say definitively, but I am happy to do so if you feel this is work exploring. It's worth noting that `template` support is unavailable for Edge 12, and all versions of Edge have [known issues](https://caniuse.com/#feat=template) that may effect medium-zoom depending on users define their template (again, haven't investigated). We can look into these if/when we tackle `template` support changes. For now, perhaps it is best to handle `template` support (and any related discussions) as a separate issue and close on these changes which provide "core" support for IE/Edge 12+13 and prevent legacy browsers from breaking. 

So, how do you feel about proceeding as follows:

1. Switch CustomEvent polyfill used in PR to a ponyfill
1. Add feature detection to allow `options.template` to be ignored in browsers that do not support HTML templates.
1. Update README to include browser support section
   - List support beginning with IE10+
   - List polyfill requirement for IE and Edge 12 for `options.template` support
1. Update README `options.support` section to include IE/Edge12 polyfill requirement

If this works for you, I can tackle these changes early next week.

Thanks!

Reply:
That sounds great, thanks for the time you put into this! I'm away next week but ping me if there's anything I can help with.

Reply:
You bet! Thanks for being amenable to the changes. :)

I'll make the updates mid next week. We can sync up again after the PR request is submitted.